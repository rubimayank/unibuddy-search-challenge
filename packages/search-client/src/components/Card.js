import styled from 'styled-components';

const Card = styled.div`
  position: relative;
  padding: 12px;
  box-shadow: rgba(0,0,0,0.12) 1px 1px 1px 2px;
  border-radius: 6px;
  > div {
    padding-bottom: 12px;
    &:last-child {
      padding-bottom: 0;
    }
  }
  margin: 8px;
`;

export default Card;
