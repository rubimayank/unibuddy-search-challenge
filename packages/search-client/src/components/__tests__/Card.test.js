import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import Card from '../Card';

describe('Card', () => {
  it('it should render', () => {
    const snap = renderer.create(<Card />).toJSON();
    expect(snap).toMatchSnapshot();
  });
});
