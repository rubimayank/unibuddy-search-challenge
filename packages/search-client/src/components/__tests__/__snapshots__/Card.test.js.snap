// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card it should render 1`] = `
.c0 {
  position: relative;
  padding: 12px;
  box-shadow: rgba(0,0,0,0.12) 1px 1px 1px 2px;
  border-radius: 6px;
  margin: 8px;
}

.c0 > div {
  padding-bottom: 12px;
}

.c0 > div:last-child {
  padding-bottom: 0;
}

<div
  className="c0"
/>
`;
